source/_docs/z-wave/controllers.markdown

```diff
@@ -9,9 +9,22 @@ You need to have a compatible Z-Wave stick or module installed. The following de
 
 {% warning %}
 
-Until recently, 700 series Z-Wave Controllers had a bug that could cause the mesh to be flooded on some networks and the controller to become unresponsive. At present, all 700 series controllers share the same firmware and are subject to this bug. It appears that this bug is largely, if not completely, resolved as of firmware version 7.17.2.
-
-Users should upgrade the firmware on all 700 series controllers to version 7.17.2 or greater. Firmware can be upgraded using the below directions:
+The firmwares of 700 and 800 series Z-Wave controllers have several bugs which impact the stability of the mesh and can cause the controller to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
+- 700 series:
+  - prefer SDK versions 7.17.2 to 7.18.x
```

Comment:
Should we avoid the term SDK here and replace it with firmware version to make it less technical for the reader?

Reply:
I'd love to, but manufacturers have their own firmware versioning that's not related to the SDK versions.
e.g. Zooz has 1.10, 1.20, 1.30, 1.40. Z-Wave.me seems to have 7.x firmwares, but with different minor versions than the SDK.

Reply:
Unless we want to explicitly list the recommended firmware versions, but the warning may span half a page then.

Reply:
Hmm it might help to explain just that in that case.

Reply:
Will add an explanation